controllers/resetController.js

```diff
@@ -0,0 +1,93 @@
+import { v4 as uuid } from 'uuid';
+import { users, resetTokens } from '../models';
+import db from '../models/index.js';
+import sendMail from './resetUtil.js';
+import { success, fail, sendError } from '../function/respond.js';
+
+export const forgotPassword = async (req, res) => {
+	try {
+		const { email } = req.body;
+		if (!req.body.email) {
+			return fail(res, 401, null, 'missingEmail', req);
```

Comment:
Please use the correct status codes. 
https://developer.mozilla.org/en-US/docs/Web/HTTP/Status